kernels/depth_projection.cl

```diff
@@ -0,0 +1,41 @@
+__constant sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
+
+__kernel void depth_projection(
```

Comment:
Consider dropping this kernel. It's pretty ImageJ-specifc.

Reply:
We can, though nothing force us to do it. Lets keep it but mark it as _**deprecated**_